### PR TITLE
Document Underwater Renderer shader API

### PR DESCRIPTION
**Pull request by daleeidd**
Now that the feature is done for all RPs I have added documentation. It can be found here: https://crest.readthedocs.io/en/docs-ur-shader-api/user/underwater.html#underwater-shader-api

Let me know if you think it makes sense. I am leaning a bit on the example shaders for the shader setup but that should be fine I think.

### CODE REVIEW

File: docs/user/underwater.rst [hunk 1]
```diff
@@ -128,6 +128,80 @@ This feature also clips the ocean surface to match.
...
+
+.. only:: urp
+
+   .. tab:: `URP`
+
+      .. include:: includes/_underwater-shader-graph.rst
+
+      For best results using the `URP_Lit_Shader` graph:
+
+      -  Be mindful of what features on the *Shader Graph* you enable as it might affect the underwater fog
```
Comment (huwb) [inline docs/user/underwater.rst:202]: Maybe which features on the shader graph master node

**Review by huwb — APPROVED** [review]
Looks good just a minor clarification

File: docs/user/underwater.rst (hunk 1 above)
Comment (daleeidd) [reply docs/user/underwater.rst:202]: I will remove this as on second look there doesn't appear to be any.

Comment (daleeidd) [reply docs/user/underwater.rst]: Removed

**Comment by daleeidd** [comment]
Thanks!

**Pull request merged**